sycl/cmake/modules/BuildUnifiedRuntime.cmake

```diff
@@ -1,5 +1,4 @@
-# Either fetches UR from the appropriate repo or sets up variables based on user
-# preference.
+# Builds in-tree UR
 
 # TODO: taken from sycl/plugins/CMakeLists.txt - maybe we should handle this
 # within UR (although it is an obscure warning that the build system here
```

Comment:
nit: rm `option(SYCL_UR_USE_FETCH_CONTENT` and some other (now redundant) definitions, just down below